russian/minesweeper/minesweeper.md

```diff
@@ -0,0 +1 @@
+Первое, что вам пришло в голову - это написать зефирного сапера в зефирном браузере. Вы сели и написали его за восемь с половиной часов.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.